docs/setup-and-installing.md

```diff
@@ -1,153 +1,81 @@
-
-    1. Click the deployment’s name to open the overview page.
-
-    1. To get links to share the game with others, click **Share**.
-
-    1. Click **Launch**.
-        > You can ignore the prompt to install the Launcher, as it’s installed as part of SpatialOS.
-
-    1. Once you’ve finished playing, click **Stop** in the Console.
+### Sign up for a SpatialOS account
+
+You'll need a SpatialOS account in order to run the SpatialOS GDK for Unity.
+Follow [this link](https://improbable.io/get-spatialos) to sign up for free.
+
+### Set up your machine
+
+#### Windows
+
+1. [Unity 2018.2.8](https://unity3d.com/get-unity/download/archive) 
+    - Make sure to select the following components:
+        - **Linux Build Support**
+        - **Mac Build Support**
+2. [Visual Studio 2017](https://www.visualstudio.com/downloads/) (Verified with version `15.8.4`) or [Rider](https://www.jetbrains.com/rider/) (Verified with version `2018.2.1`)
+    - If you are using Visual Studio, within the Visual Studio Installer, on the **Workloads** tab:
+        - Select **Game development with Unity** 
+        - Select **.NET Core cross-platform development** 
+        - In the **Summary** on the right: 
+            - Deselect **Unity 2017.2 64-bit Editor** (the SpatialOS GDK for Unity requires Unity 2018.2.8)
+            - Make sure **Visual Studio Tools for Unity** is selected
+    - If you are using Rider: 
+        - Install the [**Unity Support** plugin](https://github.com/JetBrains/resharper-unity) for a better experience
+3. [.NET Core SDK (x64)](https://www.microsoft.com/net/download/) (Verified with version `2.1.3xx` or `2.1.4xx`)
+4. SpatialOS, using the the [SpatialOS installer](https://console.improbable.io/installer/download/stable/latest/win). This installs:
+    - the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/spatial-cli-introduction)
+    - the SpatialOS [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher)
+    - the 32-bit and 64-bit Visual C++ Redistributables
+5. (Optional) [ReSharper](https://www.jetbrains.com/resharper/) if you are using Visual Studio
+    - It’s useful to have ReSharper installed if you want to contribute to the SpatialOS Unity GDK repository, as it makes it easy to stick to our [coding standards](contributions/unity-gdk-coding-standards.md). 
+    - _We are currently not accepting public contributions - see our [contributions](../.github/CONTRIBUTING.md) policy._
+6. Reboot your machine to finish the setup.
+
+> **Warning**: Older versions of Visual Studio 2017 have been known to cause some issues with Unity 2018.2.8 - the issues are projects loading and unloading frequently, and Intellisense breaking. If you do experience these issues, try updating to a newer version of Visual Studio 2017.
+
+#### Mac
+
+1. [Unity 2018.2.8](https://unity3d.com/get-unity/download/archive) 
+    - Make sure to select the following components:
```

Comment:
...components during installation.

(Makes it clear where you get the components.)